README.md

```diff
@@ -2,6 +2,8 @@ analytics-php
 ==============
 [![Segmentio](https://circleci.com/gh/segmentio/analytics-php.svg?style=shield&circle-token=a20a4b1daa865d164e62ca59a65d6753d286c1c1)](https://app.circleci.com/pipelines/github/segmentio/analytics-php)
 
+[![codecov](https://codecov.io/gh/segmentio/analytics-php/branch/master/graph/badge.svg?token=804hPfMd0C)](https://codecov.io/gh/segmentio/analytics-php)
```

Comment:
Can we keep these on the same line? Same as https://github.com/segmentio/analytics.net

Reply:
Done